demos/common/export_models/export_model.py

```diff
@@ -443,6 +450,25 @@ def export_embeddings_model(model_repository_path, source_model, model_name, pre
     print("Created subconfig {}".format(os.path.join(model_repository_path, model_name, 'subconfig.json')))
     add_servable_to_config(config_file_path, model_name, os.path.relpath(os.path.join(model_repository_path, model_name), os.path.dirname(config_file_path)))
 
+def export_embeddings_model_ov(model_repository_path, source_model, model_name, precision, task_parameters, version, config_file_path, truncate=True):
+    set_max_context_length = ""
+    destination_path = os.path.join(model_repository_path, model_name, "ov")
+    print("Exporting embeddings model to ",destination_path)
+    if not os.path.isdir(destination_path) or args['overwrite_models']:
+        optimum_command = "optimum-cli export openvino --model {} --disable-convert-tokenizer --task feature-extraction --weight-format {} --trust-remote-code --library sentence_transformers {}".format(source_model, precision, destination_path)
+        if os.system(optimum_command):
+            raise ValueError("Failed to export embeddings model", source_model)
+        set_rt_info(destination_path, 'openvino_model.xml', 'config.json')
+        if truncate:
+            max_context_length = get_models_max_context(destination_path, 'config.json')
+            if max_context_length is not None:
+                set_max_context_length = "--max_length " + str(get_models_max_context(destination_path, 'config.json'))
+        print("Exporting tokenizer to ", destination_path)
+        from openvino_tokenizers import convert_tokenizer
+        convert_tokenizer_command = "convert_tokenizer -o {} {} {}".format(destination_path, source_model, set_max_context_length) 
+        if (os.system(convert_tokenizer_command)):
+            raise ValueError("Failed to export tokenizer model", source_model)
+        set_rt_info(destination_path, 'openvino_tokenizer.xml', 'tokenizer_config.json')
```

Comment:
set_rt_info is not needed. calculator is reading the config from config.json instead of rt_info
why graph is not created?